package.json

```diff
@@ -155,10 +155,11 @@
         }
     },
     "scripts": {
-        "vscode:prepublish": "tsc -p ./",
-        "compile": "tsc -watch -p ./",
-        "update-vscode": "node ./node_modules/vscode/bin/install",
-        "postinstall": "node ./node_modules/vscode/bin/install"
+        "vscode:prepublish": "npm run compile",
```

Comment:
What is the difference between `tsc -p ./` and `npm run compile`? for `prepublish`.

Reply:
Semantically nothing, it's just a bit of indirection since `npm run compile` will run `compile` which is `tsc -p ./` and then moved the watch functionality to `npm run watch`.  I just updated the `scripts` section to look a bit more like the code from a freshly generated extension.